contrib/babelfishpg_common/src/varbinary.c

```diff
@@ -34,6 +34,19 @@
 
 #include "instr.h"
 
+/* We need to create a minimal stub of PLtsql_protocol_plugin so
+ * that we can reference pltsql_protocol_plugin_ptr to check whether
+ * we are coming from the TDS or PG client. This is required to support
+ * COPY inputs coming from DMS.
+ */
+typedef struct PLtsql_protocol_plugin
+{
+	/* True if Protocol being used by client is TDS. */
+	bool		is_tds_client;
+} PLtsql_protocol_plugin;
+PLtsql_protocol_plugin **pltsql_protocol_plugin_ptr;
```

Comment:
Can this be done in a header file in common?
Also why is it a double pointer?

Reply:
As mentioned, this is a stub that is intended to recreate what is in `pltsql.h` from babelfishpg_tsql. It could be added in a header file, but I think putting it here minimizes the potential blast radius since this is technically a global variable, but I would not want it to be accessible outside of this file.

Reply:
updated to use rendezvous variable instead